packages/apps/human-app/server/src/modules/governance/governance.service.ts

```diff
@@ -10,12 +10,10 @@ import { ProposalResponse } from './model/governance.model';
 
 const N_BLOCKS_LOOKBACK = 100000;
 
-type Proposal = {
-  proposalId: string;
-  voteStart: number;
-  voteEnd: number;
+type Block = {
+  timestamp: number;
+  number: number;
 };
-
 @Injectable()
 export class GovernanceService {
   private readonly logger = new Logger(GovernanceService.name);
```

Comment:
Sorry, didn't notice in original PR: we should be using our logger now
```suggestion
  private readonly logger = logger.child({ context: GovernanceService.name });
```
and
```
import logger from '../../logger';
```

It works with current implementation because we override nest logger, but the correct way is to use our custom one